packages/react/src/components/accordion/accordion-styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled, { css } from 'styled-components';
+import { Theme } from '../../themes';
+
+export const StyledAccordionGroup = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const AccordionSectionStyled = (isExpanded: boolean, theme: Theme) => css`
+    background: ${theme.greys['colored-white']};
+    border: ${isExpanded ? `1px solid ${theme.greys['dark-grey']}` : '0'};
+    border-radius: 0 0 4px 4px;
```

Comment:
yep, it would help.
I changed all the parameters used in the accordion to make use of the variables. 
We will address the implementation for cascade/overwrite in the future.